tests/components/vesync/snapshots/test_switch.ambr

```diff
@@ -36,8 +36,53 @@
 # ---
 # name: test_switch_state[Air Purifier 131s][entities]
   list([
+    EntityRegistryEntrySnapshot({
+      'aliases': set({
+      }),
+      'area_id': None,
+      'capabilities': None,
+      'config_entry_id': <ANY>,
+      'config_subentry_id': <ANY>,
+      'device_class': None,
+      'device_id': <ANY>,
+      'disabled_by': None,
+      'domain': 'switch',
+      'entity_category': None,
+      'entity_id': 'switch.air_purifier_131s_display',
+      'has_entity_name': True,
+      'hidden_by': None,
+      'icon': None,
+      'id': <ANY>,
+      'labels': set({
+      }),
+      'name': None,
+      'options': dict({
+      }),
+      'original_device_class': None,
+      'original_icon': None,
+      'original_name': 'Display',
+      'platform': 'vesync',
+      'previous_unique_id': None,
+      'supported_features': 0,
+      'translation_key': 'display',
+      'unique_id': 'air-purifier-display',
+      'unit_of_measurement': None,
+    }),
   ])
 # ---
+# name: test_switch_state[Air Purifier 131s][switch.air_purifier_131s_display]
+  StateSnapshot({
+    'attributes': ReadOnlyDict({
+      'friendly_name': 'Air Purifier 131s Display',
+    }),
+    'context': <ANY>,
+    'entity_id': 'switch.air_purifier_131s_display',
+    'last_changed': <ANY>,
+    'last_reported': <ANY>,
+    'last_updated': <ANY>,
+    'state': 'unavailable',
```

Comment:
Why is this one unavailable? It might sound like there's a bug here?

Reply:
I think you are right. I adjusted this so it fails while we figure this out. I think a test has an issue as I own a 131 and it works. I updated some of the json files as they seemed incorrect without luck.  @iprak thoughts?